nvtabular/ops/categorify.py

```diff
@@ -238,8 +228,10 @@ def __init__(
         # Only support two kinds of multi-column encoding
         if encode_type not in ("joint", "combo"):
             raise ValueError(f"encode_type={encode_type} not supported.")
+        if encode_type == "joint" and vocabs is not None:
+            raise ValueError("Passing in vocabs is not supported with a joint encoding.")
```

Comment:
I messed up in my comment there - the problem is with 'combo' encodings not joint (like combo is the feature crosses here that we don't support - joint is all good). 

```suggestion
        if encode_type == "combo" and vocabs is not None:
            raise ValueError("Passing in vocabs is not supported with a combo encoding.")
```